src/v0/util/facebookUtils/networkHandler.js

```diff
@@ -237,13 +259,19 @@ const errorResponseHandler = (destResponse) => {
     return;
   }
   const { error } = response;
-  const { status, errorMessage, tags: errorStatTags } = getStatus(error);
+  const { status, errorMessage, stats: errorStatTags } = getStatus(error);
+  if (
+    isDefinedAndNotNull(errorStatTags) &&
+    errorStatTags?.[TAG_NAMES.ERROR_TYPE] === 'accessTokenExpired'
+  ) {
+    throw new ConfigurationAuthError(`${errorMessage}`);
```

Comment:
errorMessage is already a string, we don't need to use interpolated strings

Reply:
updated

Reply:
Did you do a commit already ? I am not able to see the commit